package.json

```diff
@@ -342,7 +342,7 @@
   },
   "dependencies": {
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
-    "@glif/filecoin-wallet-provider": "2.0.0-alpha.14",
+    "@glif/filecoin-wallet-provider": "^2.0.0-beta.11",
```

Comment:
any reason not to pin to a specific version here?

Reply:
pinned, thanks!